index.js

```diff
@@ -32,14 +32,15 @@ class SupervisorExtension extends cli.Extension {
         }
 
         const service = template(fs.readFileSync(path.join(__dirname, 'ghost.supervisor.template'), 'utf8'));
-
-        return ctx.instance.template(service({
+        const contents = service({
             name: fileBaseName,
             dir: process.cwd(),
-            user: 'ghost',
+            user: uid,
```

Comment:
I'm not sure this change can be made - [supervisor docs](http://supervisord.org/configuration.html#program-x-section-example)

Reply:
This part of the config is specific to supervisor, and based on the docs (linked above), the `user` option takes a username rather than uid

Reply:
Of course can't believe i missed that, been working on legacy code where uid is the username as a string. I'll make the change to fix it tonight.